pom.xml

```diff
@@ -179,6 +179,12 @@
 			<artifactId>xmlunit-assertj</artifactId>
 			<version>${xmlunit.version}</version>
 			<scope>test</scope>
+			<exclusions>
```

Comment:
Added because:

```
Dependency convergence error for org.assertj:assertj-core:2.9.0 paths to dependency are:
+-org.springframework.ws:spring-ws:3.0.4.BUILD-SNAPSHOT
  +-org.xmlunit:xmlunit-assertj:2.6.2
    +-org.assertj:assertj-core:2.9.0
and
+-org.springframework.ws:spring-ws:3.0.4.BUILD-SNAPSHOT
  +-org.assertj:assertj-core:3.9.0

```